src/partials/contacts.html

```diff
@@ -0,0 +1,66 @@
+<section class="contacts">
+    <div class="contacts__container">
+    <h2 class="contacts__title">Contacts</h2>
+    <ul class="contacts__box">
+        <li class="contacts__card">
```

Comment:
lepiej abyć użył tutaj nazwy zamiast "contacts__card" to "contacts__item"

Reply:
zrobione

Reply:
zrobione